api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -7,10 +7,459 @@ type: Object
+                    const worksheet = workbook.addWorksheet('Main sheet');
+                    exportDataGrid({
+                        component: e.component,
+                        worksheet: worksheet,
+                        customizeCell: function(options) {
+                            const excelCell = options;
+                            excelCell.font = { name: 'Arial', size: 12 };
+                            excelCell.alignment = { horizontal: 'left' };
+                        } 
+                    }).then(function() {
+                        workbook.xlsx.writeBuffer()
+                            .then(function(buffer) {
+                                saveAs(new Blob([buffer], { type: 'application/octet-stream' }), 'DataGrid.xlsx');
+                            });
+                    });
+                    e.cancel = true;
+                }
+            }
+        }
+        </script>
+
+    ##### React
+
+        <!-- tab: App.js -->
+        import React from 'react';
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+        import DataGrid, { Export } from 'devextreme-react/data-grid';
+        import { exportDataGrid } from 'devextreme/excel_exporter';
+
+        class App extends React.Component {
+            render() {
+                return (
+                    <DataGrid ...
+                        onExporting={this.onExporting}>
+                        <Export enabled={true} />
+                    </DataGrid>
+                );
+            }
+            onExporting(e) {
+                const workbook = new ExcelJS.Workbook();
+                const worksheet = workbook.addWorksheet('Main sheet');
+                exportDataGrid({
+                    component: e.component,
+                    worksheet: worksheet,
+                    customizeCell: function(options) {
+                        const excelCell = options;
+                        excelCell.font = { name: 'Arial', size: 12 };
+                        excelCell.alignment = { horizontal: 'left' };
+                    } 
+                }).then(function() {
+                    workbook.xlsx.writeBuffer()
+                        .then(function(buffer) {
+                            saveAs(new Blob([buffer], { type: 'application/octet-stream' }), 'DataGrid.xlsx');
+                        });
+                });
+                e.cancel = true;
+            }
+        }
+        export default App;
+
+    --- 
+
+The following restrictions apply when users export **DataGrid**:   
+
+- Only XLSX files are supported.
+- Excel limits the number of grouping levels to 7, while in the **DataGrid** it is unlimited.
+- Only visible columns are exported. See the [onExporting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) option description for a workaround.
```

Comment:
The workaround uses the onExporting option, which is deprecated now. There's no need to fix it in the PR, but let's create a Trello card so as not to forget to fix it later.